terraform-infra-approvals/idam-shared-infrastructure.json

```diff
@@ -1,7 +1,7 @@
 {
   "resources": [],
   "module_calls": [
-    {"source":  "git@github.com:hmcts/cnp-module-waf?ref=SIDM-2128_add_idam_waf_exclusions"}
+    {"source":  "git@github.com:hmcts/cnp-module-waf?ref=SIDM-2128_add_idam_waf_exclusions"},
+    {"source":  "git@github.com:hmcts/cnp-module-key-vault?ref=change-to-sku-name"}
```

Comment:
please use azurermv2 branch, its already whitelisted

Reply:
It doesn't work. https://github.com/hmcts/idam-shared-infrastructure/pull/50/commits/a73a7172112912fc360802253b5452c7298b11cd